json/src/spec/builtin.rs

```diff
@@ -18,6 +18,9 @@
 
 use uint::Uint;
 
+/// Fixed pricing: each call cost the same
```

Comment:
The cost is based on rounds from input, this should be reflected in the name and the docs somehow.